contrib/mesos/pkg/scheduler/meta/labels.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package meta
+
+// kubernetes api object labels
+const (
+	RolesKey = "k8s.mesosphere.io/roles"
```

Comment:
Since we're making this part of a user-facing API can we have some docs here that indicate how the scheduler uses this API to place pods?